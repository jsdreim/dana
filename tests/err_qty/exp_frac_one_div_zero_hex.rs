#![allow(unused_imports)]
use dimensional::{qty, symbols::*};

fn main() {
    let _err = qty![1.0 m^(1/0x00)];
}
