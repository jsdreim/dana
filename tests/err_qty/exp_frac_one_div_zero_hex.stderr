error: root of degree zero cannot be defined
 --> tests/err_qty/exp_frac_one_div_zero_hex.rs:5:30
  |
5 |     let _err = qty![1.0 m^(1/0x00)];
  |                              ^^^^
